fast64_internal/sm64/sm64_geolayout_writer.py

```diff
@@ -1158,23 +1158,34 @@ def processBone(fModel, boneName, obj, armatureObj, transformMatrix,
 
 	#hasDL = bone.use_deform
 	hasDL = True
-	if bone.geo_cmd == 'DisplayListWithOffset':
-		if not zeroRotation:
-			node = DisplayListWithOffsetNode(int(bone.draw_layer),
-				hasDL, mathutils.Vector((0,0,0)))
-
-			parentTransformNode = addParentNode(parentTransformNode,
-				TranslateRotateNode(1, 0, False, translate, rotate))
-
+	if bone.geo_cmd in animatableBones:
+		if bone.geo_cmd == 'CustomAnimated':
+			if not bone.fast64.sm64.custom_geo_cmd_macro:
+				raise PluginError(f'Bone "{boneName}" needs a geo command macro.')
```

Comment:
these errors (there's another one) should probably also tell which armature is involved

something like
```suggestion
				raise PluginError(f'Bone "{boneName}" in armature {armatureObj.name} needs a geo command macro.')
```

(I think `armatureObj` is in scope here? the indentation looks weird on github)

Reply:
oh yeah it is, just added!